content/docs/introduction/comparison.md

```diff
@@ -246,41 +246,36 @@ environment, then Prometheus is a good choice.
 
 ## Prometheus vs. Sensu
 
-[Sensu](https://sensuapp.org/) is broadly speaking a more modern Nagios.
+[Sensu](https://sensu.io) is a composable monitoring pipeline that can reuse existing Nagios checks.
 
 ### Scope
 
-The same general scope differences as in the case of
-[Nagios](/docs/introduction/comparison/#prometheus-vs-nagios) apply here.
+Sensu checks adhere to the [Nagios check specification](https://assets.nagios.com/downloads/nagioscore/docs/nagioscore/3/en/pluginapi.html). Sensu checks operate on two different models: publish and subscribe or standalone. It is possible to silence not just individual checks, but entire subscriptions. 
```

Comment:
Subscriptions and standalone are Sensu terminology, you want someone who knows only Nagios (or not even Nagios) to understand what this means. So say for example that they can be defined on each host, or centrally in sensu itself - which you cover in the next line. So I'd say remove everything after the link.

You should also add to Nagios that it all comes from the server.